CMakeLists.txt

```diff
@@ -849,6 +850,20 @@ IF(WITH_ORB_SLAM AND NOT G2O_FOUND)
     ENDIF(ORB_SLAM_FOUND)
 ENDIF(WITH_ORB_SLAM AND NOT G2O_FOUND)
 
+IF(WITH_CUVSLAM)
+    FIND_PACKAGE(CuVSLAM QUIET)
+    IF(CUVSLAM_FOUND)
+        MESSAGE(STATUS "Found cuVSLAM: ${CUVSLAM_INCLUDE_DIRS}")
+        # cuVSLAM requires CUDA runtime
+        find_package(CUDA REQUIRED)
+        IF(CUDA_FOUND)
+            MESSAGE(STATUS "Found CUDA for cuVSLAM: ${CUDA_INCLUDE_DIRS}")
+        ELSE()
+            MESSAGE(FATAL_ERROR "CUDA is required to build with cuVSLAM! Set -DWITH_CUVSLAM=OFF if you don't have CUDA.")
+        ENDIF()
```

Comment:
What about making this check inside `FindCuVSLAM.cmake`?

You may either do
1. a `find_package(CUDA REQUIRED)` inside `FindCuVSLAM.cmake`, and cmake will automatically exit if CUDA is not found (because of keywork REQUIRED), or
2. a `find_package(CUDA)` (not "required" to handle error ourself) and do the `IF(CUDA_FOUND)` with a warning instead of fatal, then return CUVSLAM_FOUND=NO.